actions/lib/observability.js

```diff
@@ -37,70 +37,47 @@ class ObservabilityClient {
       }
     }
 
+    severityMap = {
+      'DEBUG': 1,
+      'VERBOSE': 2,
+      'INFO': 3,
+      'WARNING': 4,
+      'ERROR': 5,
+      'CRITICAL': 6,
+    }
+
+    stateToSeverity(state) {
+      const stateToSeverityMap = {
```

Comment:
would it make sense to flag completed activations as `ERROR` in case there is at least one failed path?